### PR TITLE
[Bug] [Build] fix clang build fail

### LINKED ISSUE #9322: [Bug] [Build] clang build fail
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master

### What's Wrong?

build fail with clang

### What You Expected?

fix 

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
# Proposed changes
@HappenLee 
Issue Number: close #9322

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/vec/exec/vbroker_scan_node.h [hunk 1]
```diff
@@ -33,6 +33,7 @@ class VBrokerScanNode final : public BrokerScanNode {
     VBrokerScanNode(ObjectPool* pool, const TPlanNode& tnode, const DescriptorTbl& descs);
     ~VBrokerScanNode() override = default;
 
+    using BrokerScanNode::get_next;
```
Comment (yiguolei) [inline be/src/vec/exec/vbroker_scan_node.h:36]: add get_next method in BrokerScanNode and test whether could fix the warning.

**Review by yiguolei — APPROVED** [review]
lgtm

**Review by jackwener — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**